src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -92,22 +93,13 @@ void BaseCouplingScheme::receiveData(m2n::PtrM2N m2n, DataMap receiveData)
   for (DataMap::value_type &pair : receiveData) {
     int size = pair.second->values().size();
     if (size > 0) {
-      m2n->receive(pair.second->values().data(), size, pair.second->mesh->getID(), pair.second->getDimensions());
+      m2n->receive(pair.second->values().data(), size, pair.second->getMeshID(), pair.second->getDimensions());
```

Comment:
How did this end up here?

Reply:
Also related to https://github.com/BenjaminRodenberg/precice/commit/f5c60edb2589544934fc8881c90d3dc1a550ee9e.

Reply:
This is cleaned up now via 03585b1. Especially 173dbdf.